sigle/src/modules/layout/components/AppHeader.tsx

```diff
@@ -103,7 +108,9 @@ export const AppHeader = () => {
               width={101}
               height={45}
               objectFit="cover"
-              src="/static/img/logo.png"
+              src={`/static/img/${
+                resolvedTheme === 'dark' ? 'logo_white.png' : 'logo.png'
```

Comment:
This will create the same warning as we had in stxstats no?

Reply:
Ahh yes had forgotten about this! Updating now :)